SPECS/virtiofsd/virtiofsd.spec

```diff
@@ -16,6 +16,10 @@ Source0:        https://gitlab.com/virtio-fs/virtiofsd/-/archive/v%{version}/%{n
 #   tar -czf ../%{name}-v%{version}-cargo.tar.gz vendor/
 Source1:        %{name}-v%{version}-cargo.tar.gz
```

Comment:
You don't need this anymore. The new vendor tarball replaces this.

Reply:
Removed the old vendor tarball

Reply:
Removed the old vendor tarball